ggml/src/ggml-cuda/ssm_conv.cu

```diff
@@ -0,0 +1,100 @@
+#include "ssm_conv.cuh"
+
+template <int block_size>
+static __global__ void ssm_conv_f32(
+    const float * __restrict__ src0, const float * __restrict__ src1,
+    const int src0_nb0, const int src0_nb1, const int src0_nb2,
+    const int src1_nb1,
+    float * dst,
```

Comment:
You can also add `__restrict__` here though it won't make as much of a difference.